assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -1,75 +1,23 @@
 import { PluginSidebar, PluginSidebarMoreMenuItem } from "@wordpress/edit-post";
 import { Component } from '@wordpress/element';
-import { PanelBody, RadioControl, SelectControl } from '@wordpress/components';
-import { compose } from '@wordpress/compose';
-import ColorPaletteControl from '../ColorPaletteControl/ColorPaletteControl';
-import { withPostMeta } from '../PostMeta/withPostMeta';
-import { withDefaultLabel } from '../withDefaultLabel/withDefaultLabel';
 import { __ } from '@wordpress/i18n';
-import { fromThemeOptions, getFieldFromTheme } from '../fromThemeOptions/fromThemeOptions';
+import { resolveField } from '../fromThemeOptions/fromThemeOptions';
 import isShallowEqual from '@wordpress/is-shallow-equal';
 import { savePreviewMeta } from '../../saveMetaToPreview';
+import { PostParentLink } from './PostParentLink';
+import { LegacyThemeSettings } from './LegacyThemeSettings';
 
-const themeOptions = [
-  {
-    value: '',
-    label: 'Default',
-  },
-  {
-    value: 'antarctic',
-    label: 'Antarctic',
-  },
-  {
-    value: 'arctic',
-    label: 'Arctic',
-  },
-  {
-    value: 'climate',
-    label: 'Climate Emergency',
-  },
-  {
-    value: 'forest',
-    label: 'Forest',
-  },
-  {
-    value: 'oceans',
-    label: 'Oceans',
-  },
-  {
-    value: 'oil',
-    label: 'Oil',
-  },
-  {
-    value: 'plastic',
-    label: 'Plastics',
-  },
-
-];
-
-const ThemeSelect = withPostMeta( SelectControl );
-
-const SelectWithDefaultLabel = compose(
-  fromThemeOptions,
-  withPostMeta,
-  withDefaultLabel,
-)( SelectControl );
-
-const Select = compose(
-  fromThemeOptions,
-  withPostMeta,
-)( SelectControl );
-
-const Radio = compose(
-  fromThemeOptions,
-  withPostMeta,
-)( RadioControl );
-
-const ColorPalette = compose(
-  fromThemeOptions,
-  withPostMeta,
-)( ColorPaletteControl );
-
+const loadTheme = async( value ) => {
+  if ( value === '' || !value ) {
+    value = 'default';
+  }
+  const baseUrl = window.location.href.split( '/wp-admin' )[ 0 ];
+  const themeJsonUrl = `${ baseUrl }/wp-content/themes/planet4-master-theme/campaign_themes/${ value }.json`;
+  console.log( `fetching theme ${ value }` );
```

Comment:
Is this console log meant to stay here? 🙂

Reply:
It was already there, not sure why I preserved it last time :thinking:

Reply:
I think I kept it because I didn't trust the code would call this only when needed. Not sure if I trust it now :sweat_smile: